src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -83,9 +85,35 @@ private void LogCommonPropertiesAndNewLine(IOperation operation)
             }
 
             LogString(")");
+
+            // Syntax
+            LogString($" (Syntax: {GetSnippetFromSyntax(operation.Syntax)})");
+
             LogNewLine();
         }
 
+        private static string GetSnippetFromSyntax(SyntaxNode syntax)
+        {
+            if (syntax == null)
+            {
+                return "null";
+            }
+
+            var text = syntax.ToString().Replace("\n", Environment.NewLine);
+            var lines = text.Split(new[] { Environment.NewLine, "\r", "\n" }, StringSplitOptions.RemoveEmptyEntries).Select(l => l.Trim()).ToArray();
```

Comment:
`Replace` appears to be unnecessary since `Split` splits on both `"\n"` and `Environment.NewLine`.